module1/homework/lengthSort/sort.hpp

```diff
@@ -0,0 +1,4 @@
+#pragma once
+#include <deque>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>&);
```

Comment:
I know I shoulnt be eof-nazi so i just make here 👍  ;d (and only here) ;p

Reply:
eof-nazi :)